use-cases/predictive_maintenance/2_dataprep_predmaint.ipynb

```diff
@@ -4,114 +4,85 @@
    "cell_type": "markdown",
```

Comment:
<p>The <code>contents</code> link in this cell seems to be extraneous - not added by you, but it just links to the directory containing the notebook, so maybe should be deleted. (3 occurrences of these "contents" links throughout the notebook)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3365/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Deleted all content links in all notebooks</p>